uint8 camera_ctrl
uint8 gimbal_mode
float32 pitch
float32 yaw
float32 roll
float32 duration
